### PR TITLE
 create movie detail ui

### PR DESCRIPTION
**Pull request by meoconlontonbebong**
Link Redmi : https://edu-redmine.framgia.vn/issues/6822
 Create Detail Ui Activity , Các Fragment có trong Activity .

image

### CODE REVIEW

**Comment by meoconlontonbebong** [comment]
ready

File: app/src/main/res/drawable/ic_favorite_border_back.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+
+</selector>
```
Comment (duccnv-1684) [inline app/src/main/res/drawable/ic_favorite_border_back.xml]: enter last line

File: app/src/main/res/layout/fragment_actor.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android" android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+</android.support.constraint.ConstraintLayout>
```
Comment (duccnv-1684) [inline app/src/main/res/layout/fragment_actor.xml]: enter last line

File: app/src/main/res/layout/fragment_info.xml [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        <android.support.v7.widget.RecyclerView
+            android:id="@+id/recycler_genre"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/dp_8"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/text_movie_overview" />
+    </android.support.constraint.ConstraintLayout>
+</layout>
```
Comment (duccnv-1684) [inline app/src/main/res/layout/fragment_info.xml]: enter last line

**Comment by meoconlontonbebong** [comment]
ready

**Pull request merged**